packages/manager/modules/nasha/src/components/metrics/metrics.template.html

```diff
@@ -0,0 +1,103 @@
+<div class="nasha-components-metrics col-xs-12 well top-space-p20">
+    <div class="row">
+        <div class="col-xs-12 col-lg-7">
+            <dl class="dl-horizontal">
+                <dt
+                    data-translate="nasha_components_metrics_ip"
+                    class="bottom-space-m4"
+                ></dt>
+                <dd data-ng-bind="$ctrl.nasha.ip" class="bottom-space-m8"></dd>
+                <dt
+                    data-translate="nasha_components_metrics_datacenter"
+                    class="bottom-space-m4"
+                ></dt>
+                <dd
+                    data-translate="{{:: ('nasha_components_metrics_datacenter_' + $ctrl.nasha.datacenter) }}"
+                    class="bottom-space-m8"
+                ></dd>
+                <dt
+                    data-translate="nasha_components_metrics_capacity"
+                    class="bottom-space-m4"
+                ></dt>
+                <dd class="info-total-capacity">
+                    <nasha-components-space-meter
+                        class="cloud-space-meter"
+                        data-usage="$ctrl.nasha.use"
+                        data-legend="true"
+                        data-large="true"
+                    >
+                    </nasha-components-space-meter>
+                    <span class="info-total-capacity-text">
+                        <i
+                            data-translate="nasha_components_metrics_capacity_delay_text"
+                        ></i>
+                    </span>
+                </dd>
+            </dl>
+        </div>
+
+        <div class="col-xs-12 col-lg-5 pull-right">
+            <dl class="dl-horizontal bottom-space-m0">
```

Comment:
**suggestion**: `.oui-description` can be used to maintain the similar User experience. With this usage, `.bottom-space-m4` can also be removed.